02-express-tutorial/app.js

```diff
@@ -1 +1,53 @@
-console.log('Express Tutorial')
+// console.log('Express Tutorial')
+
+const express = require('express');
+const { products } = require('./data');
+const app = express();
+
+app.use(express.static("./public"));
+
+app.get('/api/v1/test', (req, res) => {
+    res.json({ message: 'It worked!' });
+});
+
+app.get('/api/v1/test', (req, res) => {
+    res.json(products);
+});
+
+app.get('/api/v1/test', (req, res) => {
+    res.json(req.params);
+});
```

Comment:
You should only have one handler per route and request type (i.e "GET"). Here, you've defined three.